config/runtime.exs

```diff
@@ -80,10 +80,10 @@ host = System.get_env("VIRTUAL_HOST", "localhost")
 port = 4000
 
 config :membrane_live, MembraneLive.Repo,
-  username: System.get_env("PGUSER", "swm"),
-  password: System.get_env("PGPASSWORD", "swm123"),
+  username: System.get_env("POSTGRES_USER", "swm"),
+  password: System.get_env("POSTGRES_PASSWORD", "swm123"),
   hostname: System.get_env("PGHOST", "localhost"),
```

Comment:
in `.env.sample` we have `VIRTUAL_HOST=localhost`

Reply:
`VIRTUAL_HOST` is something else but I agree PGHOST should become `POSTGRES_HOST` for consistency